data_visualization_in_open_source_software/data_visualization_in_open_source_software.md

```diff
@@ -0,0 +1,286 @@
+</div>
+****
+
+Which of the following statements is the **most** true of a typical workflow for building new data visualizations?
+
+[( )] People new to data visualization should start by copying code from example plots, but as they get better they'll be able to write plot code themselves
+[( )] It's best to plan your plot carefully from the beginning so that you can get it right the first time and don't have to re-work it
+[(X)] You should pick a plot type that is a good fit both for the types of variables you have and what idea you want your plot to communicate
+[[?]] Hint: Although all of these answers have some element of truth to them, only one is **completely** true.
+[[?]] Hint: See [data visualization workflow](#data-visualization-workflow) for a review.
+****
+<div class = "answer">
+Although the first two answers seem plausible, they don't really describe a typical workflow:
+
+- Everyone, including experts, reuses code from example plots and previous visualizations (that's why [galleries of example plots](#data-visualization-workflow) are so popular!). You shouldn't expect to be able to write `ggplot2` or `seaborn` code from a blank slate. As with so many programming skills, the main difference between a `ggplot2`/`seaborn` expert and a novice is that the expert knows exactly what to google!
+- Although planning a plot carefully before you start is a good idea and will save you time, don't expect that it will save you from having to re-work it. Creating a good visualization is almost always an iterative process --- that's one reason it's such an advantage to do your plotting in a scripted language like R or python! If you want to make a small change to a plot, you don't have to start from scratch, you can just tweak the code and re-run it.
+</div>
+****
+
+Which of the following are good strategies to extend the accessibility of your data visualizations? Select all that apply.
+ [[X]] Avoid using color as the sole means of distinguishing important information
+ [[X]] Provide informative alt text (for online visuals) or captions
+ [[X]] Avoid cluttered, complex visuals
+ [[X]] Present the information from the plot in alternate forms (a data table in an appendix, narrative description in the text, etc.)
+ ***
+<div class = "answer">
+All of these are good strategies!
+
+In general, review your work asking yourself "If the reader didn't have visual access, would this still communicate everything I intend it to?" It may not always be possible to make your work maximally accessible, but there's often a lot you can do to improve it.
+
+**Note**: We've discussed visual accessibility exclusively here because that is a common stumbling block for data visualizations, but access is much more than vision. For broader guidance on creating accessible content, see the [Web Content Accessibiility Guidelines](https://www.w3.org/WAI/WCAG21/quickref/) and [The Inclusive Learning Design Handbook](https://handbook.floeproject.org/about/).
+ </div>
+ ***
```

Comment:
```suggestion
 ****
```

this quiz q isn't rendering properly -- I think it's just the number of asterisks?